src/main/java/org/takes/facets/auth/PsToken.java

```diff
@@ -117,14 +120,19 @@ private PsToken(final SiHmac sign, final long seconds) {
         String head = "";
         while (headers.hasNext()) {
             final String nexthead = headers.next();
-            if (nexthead.trim().startsWith("Bearer")) {
+            final UncheckedText trimmed = new UncheckedText(
```

Comment:
@fanifieiev Please try to use `Text` instead of `String` here. You can still inline all this code using `PrefixOf` decorator.

Reply:
@paulodamaso OK